CombiningGame/Generators/TwoRandomOneToTenNumbersGenerator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Windows.Controls;
+
+namespace CombiningGame.Generators
+{
+    class TwoRandomOneToTenNumbersGenerator
+    {
+        public int RightAnswer;
+        public int firstNumber;
+        public int secondNumber;
+
+        public void GenerateRandomNumbers()
+        {
+            var random = new Random();
```

Comment:
Use new Random in initializator or ctor.

Reply:
?

Reply:
Don't class with some magic numbers (ex 1 to 10). You can generalize this class and pass those 2 values to constructor.